bindings/const_generator.py

```diff
@@ -56,52 +56,52 @@ def CamelCase(s):
                     'post': '\n',
                 },
                 {   'regex': r'MODE_.*',
-                    'pre': 'bitflags! {\n\tpub flags Mode : u32 {\n',
+                    'pre': 'bitflags! {{\n\tpub flags Mode : u32 {{\n',
```

Comment:
"the format below would complain that there isn't a closing curly brace."  <--- how to reproduce this "complain"?